cmd/kubelet/app/options/options.go

```diff
@@ -65,92 +44,20 @@ type KubeletServer struct {
-			HTTPCheckFrequency:           unversioned.Duration{Duration: 20 * time.Second},
-			ImageMinimumGCAge:            unversioned.Duration{Duration: 2 * time.Minute},
-			ImageGCHighThresholdPercent:  90,
-			ImageGCLowThresholdPercent:   80,
-			LowDiskSpaceThresholdMB:      256,
-			MasterServiceNamespace:       api.NamespaceDefault,
-			MaxContainerCount:            240,
-			MaxPerPodContainerCount:      2,
-			MaxOpenFiles:                 1000000,
-			MaxPods:                      110,
-			NvidiaGPUs:                   0,
-			MinimumGCAge:                 unversioned.Duration{Duration: 1 * time.Minute},
-			NetworkPluginDir:             "/usr/libexec/kubernetes/kubelet-plugins/net/exec/",
-			NetworkPluginName:            "",
-			NonMasqueradeCIDR:            "10.0.0.0/8",
-			VolumePluginDir:              "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
-			NodeStatusUpdateFrequency:    unversioned.Duration{Duration: 10 * time.Second},
-			NodeLabels:                   make(map[string]string),
-			OOMScoreAdj:                  int32(qos.KubeletOOMScoreAdj),
-			LockFilePath:                 "",
-			ExitOnLockContention:         false,
-			PodInfraContainerImage:       GetDefaultPodInfraContainerImage(),
-			Port:                             ports.KubeletPort,
-			ReadOnlyPort:                     ports.KubeletReadOnlyPort,
-			RegisterNode:                     true, // will be ignored if no apiserver is configured
-			RegisterSchedulable:              true,
-			RegistryBurst:                    10,
-			RegistryPullQPS:                  5.0,
-			KubeletCgroups:                   "",
-			ResolverConfig:                   kubetypes.ResolvConfDefault,
-			RktPath:                          "",
-			RktAPIEndpoint:                   rkt.DefaultRktAPIServiceEndpoint,
-			RktStage1Image:                   "",
-			RootDirectory:                    defaultRootDir,
-			RuntimeCgroups:                   "",
-			SerializeImagePulls:              true,
-			SeccompProfileRoot:               filepath.Join(defaultRootDir, "seccomp"),
-			StreamingConnectionIdleTimeout:   unversioned.Duration{Duration: 4 * time.Hour},
-			SyncFrequency:                    unversioned.Duration{Duration: 1 * time.Minute},
-			SystemCgroups:                    "",
-			ReconcileCIDR:                    true,
-			ContentType:                      "application/vnd.kubernetes.protobuf",
-			KubeAPIQPS:                       5.0,
-			KubeAPIBurst:                     10,
-			ExperimentalFlannelOverlay:       experimentalFlannelOverlay,
-			OutOfDiskTransitionFrequency:     unversioned.Duration{Duration: 5 * time.Minute},
-			HairpinMode:                      componentconfig.PromiscuousBridge,
-			BabysitDaemons:                   false,
-			EvictionPressureTransitionPeriod: unversioned.Duration{Duration: 5 * time.Minute},
-			PodsPerCore:                      0,
-		},
+		AuthPath:             util.NewStringFlag("/var/lib/kubelet/kubernetes_auth"), // deprecated
```

Comment:
I'll move `SystemReserved` and `KubeReserved` into the config struct today. I talked to @mikedanese about these fields last week and IIRC the reason not to move `AuthPath` and `KubeConfig` into the config struct (yet) is that they are read from disk, but we do not yet have the option to read the kubelet config struct from disk. @mikedanese please correct me if I am not remembering correctly.